pulsar-testclient/src/main/java/com/yahoo/pulsar/testclient/PerformanceConsumer.java

```diff
@@ -84,7 +84,7 @@
 
         @Parameter(names = { "-c",
                 "--max-connections" }, description = "Max number of TCP connections to a single broker")
-        public int maxConnections = 0;
+        public int maxConnections = -1;
```

Comment:
It is already set that if you pass 0 it will disable the connection pool mechanism. Otherwise the client-lib default would be 1. So, I would say to set the default to 1 in this tool, so that it is more explicit what the resulting number of connection will be.